hdfs3/tests/test_hdfs3.py

```diff
@@ -159,6 +159,15 @@ def test_write_blocksize(hdfs):
         hdfs.open(a, 'rb', block_size=123)
 
 
+def test_write_vbig(hdfs):
+    with hdfs.open(a, 'wb') as f:
+        f.write(b' ' * 2**31)
```

Comment:
See https://pytest.org/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option

Or copy from `dask/conftest.py` and note the use of `--runslow` in `dask/.travis.ci`